api/src/main/resources/workbench.yaml

```diff
@@ -960,6 +960,48 @@ paths:
           description: A list of resources order by last access time desc order
           schema:
             $ref: "#/definitions/RecentResourceResponse"
+
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/user-recent-resource/updateEntry:
```

Comment:
I would add an s: user-recent-resources here and below to match the above GET API